### PR TITLE
[dev-v2.8] Add k8s v1.27 support to aks-operator 

### PR DESCRIPTION
**Pull request by furkatgofurov7**

### CODE REVIEW

File: packages/rancher-aks-operator/rancher-aks-operator-crd/package.yaml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-url: https://github.com/rancher/aks-operator/releases/download/v1.1.2/rancher-aks-operator-crd-1.1.2.tgz
+url: https://github.com/rancher/aks-operator/releases/download/v1.2.0-rc1/rancher-aks-operator-crd-1.2.0-rc1.tgz
 version: 103.0.0
-doNotRelease: true
```
Comment (rohitsuse) [inline packages/rancher-aks-operator/rancher-aks-operator-crd/package.yaml:3]: In the next rc or unrc update of chart. Remove this `doNotRelease:false` line

**Review by rohitsakala — APPROVED** [review]

**Review by mjura — COMMENTED** [review]
lgtm

File: packages/rancher-aks-operator/rancher-aks-operator-crd/package.yaml (hunk 1 above)
Comment (furkatgofurov7) [reply packages/rancher-aks-operator/rancher-aks-operator-crd/package.yaml:3]: @rohitsakala it is removed already, you meant it should be added back next time?

Comment (rohitsuse) [reply packages/rancher-aks-operator/rancher-aks-operator-crd/package.yaml:3]: It is still there https://github.com/rancher/charts/blob/dev-v2.8/packages/rancher-aks-operator/rancher-aks-operator-crd/package.yaml#L3. I meant remove it

**Pull request merged**